samples/wfi32iot/basic-sample/src/iotconnect_app.c

```diff
@@ -217,44 +202,162 @@ static void publish_telemetry() {
     button_press_data.flag.sw1 = 0;
     button_press_data.flag.sw2 = 0;
     
-    if (ULTRALOWPRESS_status == ULTRALOWPRESS_OK && ULTRALOWPRESS_isReady()) {
-        ULTRALOWPRESS_clearStatus();
-        iotcl_telemetry_set_number(msg, "ULP_temperature", ULTRALOWPRESS_getTemperature());
-        iotcl_telemetry_set_number(msg, "ULP_pressure", ULTRALOWPRESS_getPressure());            
+    //Air Quality 7 Click Reading
+    if(connections[0] == 1)
```

Comment:
Curly brace needs to be in the same line as if. The rest of this code style needs to follow the existing pattern (K&R with exceptions derfined in iotc-c-lib "contributing" guidelines)